src/Bicep.Core.Samples/Files/Parameters_LF/main.json

```diff
@@ -111,6 +111,32 @@
       "maxLength": 24,
       "minLength": 3
     },
+    "fizzArray": {
```

Comment:
Please could you also add a test case for the specific issue reported to `ScenarioTests`? With baselines, subtle changes like this could be very easily missed (especially without the context for why the thing being tested is important)